src/bin/build_octree.rs

```diff
@@ -29,10 +29,19 @@ struct CommandlineArguments {
     /// This decides on the number of bits used to encode each node.
     #[structopt(long = "resolution", default_value = "0.001")]
     resolution: f64,
+    // Flag to skip bbox calculation and use a fixed bbox.
+    // The fixed bbox is extends 1048576 length units in each direction.
+    #[structopt(long = "fixed_bbox")]
```

Comment:
How about using
```rust
use structopt::clap::{_clap_count_exprs, arg_enum};

arg_enum! {
    #[derive(Debug)]
    pub enum RootBbox {
        FromData,
        EarthECEF,
    }
}
```
in `src/generation.rs` and here
```rust
#[structopt(
    long = "bbox_type",
    raw(possible_values = "&RootBbox::variants()", case_insensitive = "true"),
    default_value = "FromData"
)]
bbox_type: RootBbox,
...
build_octree_from_file(
    &pool,
    args.output_directory,
    args.resolution,
    args.input,
    args.bbox_type,
);
```
so we get the enum instance directly?